src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1410,7 +1410,7 @@ public void ModuleManifest()
             Assert.Equal("blah", args.Win32Manifest);
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/58077")]
```

Comment:
@jaredpar pretty bad to lose all the validation of this method.  should we comment out that portion taht is failing instead?

Reply:
I also think it is better to disable specific problematic portion. Perhaps only on Linux DEBUG.